tasks/start.js

```diff
@@ -3,13 +3,40 @@
 var childProcess = require('child_process');
 var electron = require('electron-prebuilt');
 var gulp = require('gulp');
+var extend = require('util')._extend;
+var utils = require('./utils');
+
+var buildPath = './build';
 
 gulp.task('start', ['build', 'watch'], function () {
-    childProcess.spawn(electron, ['./build'], {
-        stdio: 'inherit'
-    })
-    .on('close', function () {
-        // User closed the app. Kill the host process.
-        process.exit();
+
+    utils.initBrowserSync({port: 3002}, function(bs) {
```

Comment:
I originally had everything in start.js, but it seemed a bit clunky. I leaned toward moving things to a separate file (in this case the existing utils.js). If you don't mind a bunch more code in start.js, I'm happy to move those functions there and simplify a bit.